elements/pfe-modal/demo/pfe-modal.html

```diff
@@ -5,115 +5,290 @@
-        qui officia deserunt mollit anim id est laborum.</p>
-      <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Donec sed odio dui. Sed posuere
-        consectetur est at lobortis. Donec sed odio dui. Donec ullamcorper nulla non metus auctor fringilla.</p>
-      <p>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Cum sociis natoque penatibus et magnis dis
-        parturient montes, nascetur ridiculus mus. Curabitur blandit tempus porttitor. Lorem ipsum dolor sit amet,
-        consectetur adipiscing elit. Curabitur blandit tempus porttitor. Duis mollis, est non commodo luctus, nisi
-        erat porttitor ligula, eget lacinia odio sem nec elit. Curabitur blandit tempus porttitor.</p>
-      <p>Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Donec sed
-        odio dui. Maecenas faucibus mollis interdum. Fusce dapibus, tellus ac cursus commodo, tortor mauris
-        condimentum nibh, ut fermentum massa justo sit amet risus.</p>
-      <pfe-cta>
-        <!-- TODO: find out why this gets on=light, should be on=dark b/c closest modal is color-palette=darkest -->
-        <a href="#bar">Learn more</a>
-      </pfe-cta>
-    </pfe-modal>
-  </pfe-card>
+<pfe-band color-palette="lightest" size="small">
+  <h2 slot="header">Custom icon</h2>
+  <pfe-modal aria-describedby="custom-modal-description">
+    <h2 slot="header">
+      <pfe-icon role="presentation" icon="rh-icon-megaphone" size="1x"></pfe-icon>
+      Modal Header
+    </h2>
+    <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
+      aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+      occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <pfe-button slot="trigger">
+      <button>Show modal</button>
+    </pfe-button>
+  </pfe-modal>
+</pfe-band>
+
+<pfe-band color-palette="lightest" size="small">
+  <h2 slot="header">Warning alert</h2>
+  <pfe-modal aria-describedby="custom-modal-description">
+    <h2 slot="header">
+      <pfe-icon role="presentation" icon="web-icon-alert-warning" color="moderate" size="1x"></pfe-icon>
+      Modal Header
+    </h2>
+    <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
+      aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+      occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <pfe-button slot="trigger">
+      <button>Show modal</button>
+    </pfe-button>
+  </pfe-modal>
+</pfe-band>
+
+<!-- TODO: wizard -->
+<!-- TODO: dropdown that can break out -->
+<!-- TODO: help popover -->
+<!-- TODO: with form fields -->
+
+
+<pfe-band color-palette="lightest" size="small">
```

Comment:
This comment goes along with the comment added to the *.scss about fixing the overflow content example.

```Suggestion
<pfe-band color-palette="lightest" size="small" style="--pf-c-modal-box--MaxWidth: 320px; --pf-c-modal-box--Width: 100%;">
```

Reply:
When I was testing I preferred to just manipulate the viewport. PF4 demos don't have any controls or built in styles for that. Is that enough for you?

Reply:
Works for me.